 SoftVideoDecoderOMXComponent: max-size can be less than size

A component can be configured for size that is greater than the
max size. In such case, configure for the larger of these.

diff --git a/media/libstagefright/omx/SoftVideoDecoderOMXComponent.cpp b/media/libstagefright/omx/SoftVideoDecoderOMXComponent.cpp
index 10116d6..86887b0 100644
--- a/media/libstagefright/omx/SoftVideoDecoderOMXComponent.cpp
+++ b/media/libstagefright/omx/SoftVideoDecoderOMXComponent.cpp
@@ -182,11 +182,11 @@ void SoftVideoDecoderOMXComponent::updatePortDefinitions(bool updateCrop, bool u
 
 
 uint32_t SoftVideoDecoderOMXComponent::outputBufferWidth() {
-    return max(mIsAdaptive ? mAdaptiveMaxWidth : 0, mWidth);
+    return mIsAdaptive ? mAdaptiveMaxWidth : mWidth;
 }
 
 uint32_t SoftVideoDecoderOMXComponent::outputBufferHeight() {
-    return max(mIsAdaptive ? mAdaptiveMaxHeight : 0, mHeight);
+    return mIsAdaptive ? mAdaptiveMaxHeight : mHeight;
 }
 
 void SoftVideoDecoderOMXComponent::handlePortSettingsChange(
